### ISSUE TITLE
refactoring the code

### ISSUE THREAD

**Issue by ElmekaouiHaitham** [j-j-gajjar/Mathematics#12]
the functions when generating a PDF have low cohesion and are not readable.

**Comment by j-j-gajjar** [comment]
You are correct, I created this project in the early stages of learning flutter, therefore it has a lot of complicated code!

I'm getting ready to rework this project's logic! As soon as I get some free time, I will rebuild the entire project.

Thank you for your interest in this project.

**Comment by ElmekaouiHaitham** [comment]
I made the changes, How can I push them?
